internal/constants/constants.go

```diff
@@ -5,4 +5,5 @@ const (
 	NodeLabelerFinalizer = "kmm.node.kubernetes.io/node-labeler"
 	TargetKernelTarget   = "kmm.node.kubernetes.io/target-kernel"
 	DaemonSetRole        = "kmm.node.kubernetes.io/role"
+	BuildStage           = "kmm.node.kubernetes.io/build-stage"
```

Comment:
How about `kmm.node.kubernetes.io/job-type`?

Reply:
actually BuildStage is used as a variable name in the metrics package (it doesn't conflict, but will confuse), so I've changed everything to constants.JobType and `kmm.node.kubernetes.io/job-type`